### PR TITLE
MH-247

### PR DESCRIPTION
**Pull request by iacobfred**
__Edit from @mattfredericksen__:

`/occurrences` initial load size reduced by 84%.

> image

`/quotes` reduced by 92%.

> image

Etc.

### CODE REVIEW

File: frontend/pages/entities/index.tsx [hunk 1]
```diff
@@ -26,9 +25,7 @@ const Entities: FC<EntitiesProps> = ({ entitiesData }: EntitiesProps) => {
     <Grid item key={entity.slug} xs={6} sm={4} md={3}>
       <Link href={`/entities/${entity.slug}`}>
         <a>
-          <ModuleCard module={entity} header={entity.name}>
-            <HTMLEllipsis unsafeHTML={entity.description} maxLine="4" basedOn="words" />
-          </ModuleCard>
+          <ModuleUnionCard module={entity} />
```
Comment (mattfredericksen) [inline frontend/pages/entities/index.tsx:28]: The `children` here are ignored by `ModuleUnionCard`, so I removed them.

**Comment by mattfredericksen** [comment]
@alashow any concerns with the `list_fields` implementation?

File: core/models/model.py [hunk 2]
```diff
@@ -262,6 +263,26 @@ class DrfModelSerializer(serializers.ModelSerializer):
 
     model = serializers.SerializerMethodField()
 
+    def __init__(self, *args, **kwargs):
+        # Don't pass the 'fields' arg up to the superclass
+        fields = kwargs.pop('fields', None)
```
Comment (alashow) [inline core/models/model.py:268]: Nit: Can we rename `fields` arg to `requested_fields` for clarity?

**Review by alashow — APPROVED** [review]
LGTM!

File: core/api/views.py [hunk 3]
```diff
@@ -1,13 +1,20 @@
+from typing import Optional
+
 from rest_framework.viewsets import ModelViewSet
 
 
 class ExtendedModelViewSet(ModelViewSet):
     """A ModelViewSet which allows accessing objects by slug or primary key."""
 
     lookup_url_kwarg = 'pk_or_slug'
+    list_fields: Optional[list[str]] = ['model', 'slug']
```
[the remark this answers is not in the document]
Comment (alashow) [reply core/api/views.py]: I prefer current version too.

File: core/models/model.py (hunk 2 above)
Comment (mattfredericksen) [reply core/models/model.py]: dda7b33142e2087fa88ef5ef04fefb1334ea72e5

**Pull request merged**